dirty_cat/test/test_reproducibility.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
```

Comment:
The convention is usually that tests go in a file that has the same name as the file containing the functionality that they test, but with "test_" before.